### PR TITLE
Adding Build/x features highlights to the overview

### PR DESCRIPTION
**Pull request by dockertopia**
### Proposed changes

Adding Build/x features highlights to the overview as per google doc collaborated on.

### CODE REVIEW

**Review by jedevc — CHANGES_REQUESTED** [review]
LGTM :tada:

A minor nit: there's a few extra blank lines, just for consistency we should keep to a single blank line for when we need the spacing between paragraphs/list-items/etc.

**Review by crazy-max — CHANGES_REQUESTED** [review]
Yes like @jedevc said, remove extra blank lines, otherwise LGTM

Also check invalid links: https://github.com/docker/docker.github.io/runs/7498507277?check_suite_focus=true#step:4:570

**Comment by dockertopia** [comment]
> LGTM 🎉
> 
> A minor nit: there's a few extra blank lines, just for consistency we should keep to a single blank line for when we need the spacing between paragraphs/list-items/etc.

Sure. I can't get a line break for some reason, so hence the extra lines. But sure can remove them cc: @crazy-max .

**Comment by dockertopia** [comment]
> Also check invalid links: https://github.com/docker/docker.github.io/runs/7498507277?check_suite_focus=true#step:4:570
Checking how to get these validations locally if possible.

**Comment by crazy-max** [comment]
> Checking how to get these validations locally if possible.

Type `make htmlproofer` to check links locally.

File: build/index.md [hunk 1]
```diff
@@ -27,3 +27,63 @@ BuildKit is the backend evolution from the Legacy Builder, it comes with new
 and much improved functionality that can be powerful tools for improving your
 builds' performance or reusability of your Dockerfiles, and it also introduces
 support for complex scenarios.
+
+
```
Comment (crazy-max) [inline build/index.md:31]:
```suggestion

```

File: build/index.md [hunk 2]
```diff
@@ -27,3 +27,63 @@ BuildKit is the backend evolution from the Legacy Builder, it comes with new
...
+Automate your image builds to run in GitHub actions using the official docker build actions. See:  
+    * [GitHub Action to build and push Docker images with Buildx](https://github.com/docker/build-push-action).
+    * [GitHub Action to extract metadata from Git reference and GitHub events](https://github.com/docker/metadata-action/).
+
+* **Orchestrating builds across complex projects together**  
+Connect your builds together and easily parameterize your images using buildx bake.  
+<!--replace when Bake content is onboarded -->
+See [High-level build options](buildx/index.md/#high-level-build-options)
+
+
```
Comment (crazy-max) [inline build/index.md:70]:
```suggestion

```

File: build/index.md [hunk 3]
```diff
@@ -27,3 +27,63 @@ BuildKit is the backend evolution from the Legacy Builder, it comes with new
...
+### Customizing your Builds
+
+* **Select your build output format**  
+Choose from a variety of available output formats, to export any artifact you like from BuildKit, not just docker images.  
+See [Set the export action for the build result](https://github.com/docker/buildx/blob/master/docs/reference/buildx_build.md#-set-the-export-action-for-the-build-result--o---output).
```
Comment (crazy-max) [inline build/index.md:75]: Don't link to repo docs but internal docs

File: build/index.md (hunk 2 above)
Comment (dockertopia) [reply build/index.md:70]: Removed additional empty lines which do not render.

File: build/index.md (hunk 1 above)
Comment (dockertopia) [reply build/index.md:31]: Removed additional empty lines which do not render.

File: build/index.md (hunk 3 above)
Comment (dockertopia) [reply build/index.md:75]: This was the link in our working google doc we reviewed. Linking to equivalent content found by search.

Comment (crazy-max) [reply build/index.md:75]: Sorry hard to catch on Google docs without source code.

Comment (dockertopia) [reply build/index.md:75]: Hope it's all fine now :). Could you PTAL? @crazy-max

File: build/index.md (hunk 1 above)
Comment (crazy-max) [reply build/index.md:31]: Does not look solved to me, still two new lines.

File: build/index.md (hunk 2 above)
Comment (crazy-max) [reply build/index.md:70]: Same as above

**Review by aevesdocker — APPROVED** [review]

File: build/index.md (hunk 1 above)
Comment (crazy-max) [reply build/index.md:31]: @dockertopia ^

**Review by crazy-max — CHANGES_REQUESTED** [review]
two small nits to fix, see my comments

**Review by jedevc — APPROVED** [review]

**Comment by crazy-max** [comment]
As discussed we are going to see how to tackle linting issue in #15227. I still think it's important to have proper and consistent code base, even in markdown.

**Pull request merged**